x-pack/plugins/stack_connectors/public/connector_types/slack_api/slack_connectors.tsx

```diff
@@ -36,20 +42,143 @@ const getSecretsFormSchema = (docLinks: DocLinksStart): SecretsFieldSchema[] =>
   },
 ];
 
-const NO_SCHEMA: ConfigFieldSchema[] = [];
+interface AllowedChannels {
+  id: string;
+  name: string;
+}
 
+const getConfigFormSchemaAfterSecrets = ({
+  options,
+  isLoading,
+  isDisabled,
+  onChange,
+  onCreateOption,
+  selectedOptions,
+}: {
+  options: Array<EuiComboBoxOptionOption<AllowedChannels>>;
+  isLoading: boolean;
+  isDisabled: boolean;
+  onChange: (options: Array<EuiComboBoxOptionOption<AllowedChannels>>) => void;
+  onCreateOption: (searchValue: string, options: EuiComboBoxOptionOption[]) => void;
+  selectedOptions: Array<EuiComboBoxOptionOption<AllowedChannels>>;
+}): ConfigFieldSchema[] => [
+  {
+    id: 'allowedChannels',
+    isRequired: true,
+    label: i18n.ALLOWED_CHANNELS,
+    helpText: (
+      <FormattedMessage
+        id="xpack.stackConnectors.components.slack_api.allowedChannelsText"
+        defaultMessage="By default, the connector can access all channels within the scope of the Slack app."
```

Comment:
>        defaultMessage="By default, the connector can access all channels within the scope of the Slack app."
If channel Ids is now *requred*, I think this message about the default behaviour is no longer correct and should be removed.

Reply:
Do we want to put something else?

Reply:
Unless folks think it's self-evident, we could use text like this instead: "The connector can access only these channels." IMO that's pretty obvious so I'd omit it.